google-cloud-tools-plugin/build.gradle

```diff
@@ -38,7 +38,7 @@ dependencies {
     compile(project(':common-lib'))
 
     compile files('lib/google-api-services-clouddebugger-v2-rev4-1.20.0.jar')
-    compile files('lib/google-api-services-developerprojects-v1-rev20140815212553-1.19.0.jar')
+    compile files('lib/google-api-services-cloudresourcemanager-v1beta1-rev12-1.21.0.jar')
```

Comment:
I tried quickly, and the version in Maven Central didn't work out of the box. Need to look more into it...